### PR TITLE
feat: stream manager debugging support

### PR DESCRIPTION
**Pull request by brnaba-aws**
**Issue #, if available:**

**Description of changes:**
Adding capability to interact with StreamManager from the local debug console UI. This new capability allows developers to create, delete, update and append messages to streams.

**Why is this change necessary:**
Being able gather information about streams, configuration, messages and definition helps developer to debug their components which interacts with stream manager. 

**How was this change tested:**
Tested on Greengrass as a trusted plugin.

**Any additional information or context required to review the change:**

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/main/js/dashboard-frontend/src/components/details/DetailBody.tsx [hunk 1]
```diff
@@ -85,11 +85,11 @@ export default class DetailBody extends Component<
         <ColumnLayout columns={2} variant="text-grid">
           <SpaceBetween size={"l"}>
             <div>
-              <Box margin={{bottom: "xxxs"}} color="text-label">Component name</Box>
+              <Box margin={{bottom: "xxxs"}} variant="awsui-key-label"  color="text-label">Component name</Box>
```
Comment (MikeDombo) [inline src/main/js/dashboard-frontend/src/components/details/DetailBody.tsx:88]: why is this needed?

File: src/main/js/dashboard-frontend/src/components/details/DetailBody.tsx [hunk 2]
```diff
@@ -99,15 +99,15 @@ export default class DetailBody extends Component<
...
+              <Box margin={{bottom: "xxxs"}} variant="awsui-key-label"  color="text-label">Version</Box>
               <div>{this.state.service.version}</div>
             </div>
             <div>
               <Popover
                 content='Authorship of the component. "Built-In" Components are provided with Greengrass. "User" components are ones you create.'
                 header={"Component Origin"}
               >
-                Origin
+                <b>Origin</b>
```
Comment (MikeDombo) [inline src/main/js/dashboard-frontend/src/components/details/DetailBody.tsx:110]: why change this?

**Review by MikeDombo — APPROVED** [review]

File: src/main/js/dashboard-frontend/src/components/details/DetailBody.tsx (hunk 2 above)
Comment (brnaba-aws) [reply src/main/js/dashboard-frontend/src/components/details/DetailBody.tsx:110]: Strange, I don't remember changing this. Not needed for sure.

File: src/main/js/dashboard-frontend/src/components/details/DetailBody.tsx (hunk 1 above)
Comment (brnaba-aws) [reply src/main/js/dashboard-frontend/src/components/details/DetailBody.tsx:88]: This has been suggested by YingYing, in order to have the label in bold.

Comment (MikeDombo) [reply src/main/js/dashboard-frontend/src/components/details/DetailBody.tsx:88]: Commented on the wrong thing?

Comment (brnaba-aws) [reply src/main/js/dashboard-frontend/src/components/details/DetailBody.tsx:88]: no, no this variant is the recommendation from cloudscape ambassador.
See here: https://cloudscape.design/components/key-value-pairs/

**Review by MikeDombo — APPROVED** [review]

**Review by saranyailla — APPROVED** [review]
